### ISSUE TITLE
packed_simd no longer compiles as of nightly-2020-09-16 

### ISSUE THREAD

**Issue by CjS77** [dalek-cryptography/curve25519-dalek#337]
`packed_simd` fails compilation as of nightly-2020-09-16 with macro failures, this being the first of many:

```
error[E0432]: unresolved import `crate::arch::x86_64::__m64`
  --> ~/.cargo/registry/src/github.com-1ecc6299db9ec823/packed_simd-0.3.3/src/api/into_bits/arch_specific.rs:51:15
   |
51 |               $($arch_ty),*
   |                 ^^^^^^^^ no `__m64` in `arch::x86_64`
...
86 | / impl_arch!(
87 | |     [x86["x86"]: __m64], [x86_64["x86_64"]: __m64],
88 | |     [arm["arm"]: int8x8_t, uint8x8_t, poly8x8_t, int16x4_t, uint16x4_t,
89 | |      poly16x4_t, int32x2_t, uint32x2_t, float32x2_t, int64x1_t,
...  |
96 | |     test: test_v64
97 | | );
   | |__- in this macro invocation
   |
   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
```

As per the comment in the `packed_simd` [README](https://github.com/rust-lang/packed_simd):

> The Crates.io Version Can No Longer Be Updated!
 The original maintainer is out of contact, and the new maintainers (the Portable SIMD Project Group) do not have the appropriate crates.io permissions to issue updates.
We are aware that the version available on crates.io is currently broken, and will not build.
If you need to continue to use the crate, we have published a "next version" under an alternative name.
Adjust your [dependencies] section of Cargo.toml to be the following:

    packed_simd = { version = "0.3.4", package = "packed_simd_2" }

I've tested this change locally and can confirm that the package compiles and tests pass with this alias.

**Comment by MarkusZoppelt** [comment]
We are aware of that issue. Please check #333.

There is also a fix in the [develop](https://github.com/dalek-cryptography/curve25519-dalek/tree/develop) branch after PR #334

**Comment by isislovecruft** [comment]
Fixed in versions 3.0.1, 2.1.1, and 1.2.5.
